src/analysis/normed_space/basic.lean

```diff
@@ -619,7 +619,7 @@ begin
   rwa [dist_comm, dist_eq_norm, add_sub_cancel'],
 end
 
-lemma tendsto_inv [normed_field α] {r : α} (r0 : r ≠ 0) : tendsto (λq, q⁻¹) (𝓝 r) (𝓝 r⁻¹) :=
+lemma tendsto_inv' [normed_field α] {r : α} (r0 : r ≠ 0) : tendsto (λq, q⁻¹) (𝓝 r) (𝓝 r⁻¹) :=
```

Comment:
I don't think you need to add a ' on this lemma and the next one, they are in the `normed_field` namespace so they wouldn't clash with anything else

Reply:
Ok, and I guess it's the same for `continuous_on_inv` ?